app/src/main/java/de/westnordost/streetcomplete/overlays/surface/SurfaceColorMapping.kt

```diff
@@ -0,0 +1,21 @@
+package de.westnordost.streetcomplete.overlays.surface
+
+import de.westnordost.streetcomplete.osm.surface.Surface
+import de.westnordost.streetcomplete.osm.surface.Surface.*
+import de.westnordost.streetcomplete.overlays.Color
+
+val Surface?.color get() = when (this) {
+    // https://davidmathlogic.com/colorblind/#%23444444-%23FF0000-%231A87E6-%232FACE8-%2330D4EE-%2310C1B8-%230DA082-%23F37D1E-%23EEBD0D-%23B6EF28-%23DDDDDD-%23999999
+    ASPHALT, CONCRETE, PAVING_STONES, PAVING_STONES_WITH_WEIRD_SUFFIX, BRICK, BRICKS, WOOD, METAL -> Color.BLUE
+    CONCRETE_PLATES, CONCRETE_LANES, SETT, COBLLESTONE_FLATTENED -> Color.SKY
+    COMPACTED -> Color.CYAN
+    UNHEWN_COBBLESTONE -> Color.AQUAMARINE
+    GRASS_PAVER -> Color.TEAL
+    SAND  -> Color.ORANGE
+    GRASS  -> Color.LIME
+    DIRT, SOIL, EARTH, MUD, GROUND_ROAD, GROUND_AREA, WOODCHIPS -> Color.GOLD
+    FINE_GRAVEL -> "#dddddd"
+    GRAVEL, PEBBLES, ROCK -> "#999999"
```

Comment:
Yeah, that's not nice. However, I promised you a "greyscale" mode, but I found out that this does not work well. (It doesn't look good because all the colors used have a very similar brightness + is a PITA to implement).

So, I guess nothing speaks against adding grey to the color palette. However, in this case here, maybe add some rationale (as comment) why you chose these colors. My gut feeling tells me that the most used and thus most uninteresting color should be used for asphalt + paving stones, so that it does not stand out. But anyway, succinct comment regarding the choices would be nice. (Is it roughly blueish = good, orangeish = bad, maybe?)

Reply:
> My gut feeling tells me that the most used and thus most uninteresting color should be used for asphalt + paving stones, so that it does not stand out.

I tried that and cities were looking quite depressing with overlay enabled. I intentionally tried something more interesting.